src/client/src/core/util/validation/rules.ts

```diff
@@ -690,3 +690,14 @@ export const validVideoURL = function (value: string): boolean | string {
     translate('general.invalid_video_url_message')
   );
 };
+
+export const digitsOnlyWithTwoDecimalPoints = function (
+  value: string,
+): boolean | string {
+  return (
+    value == '' ||
+    /^\d+\.\d{2}$/.test(value) ||
```

Comment:
we don't need 2 regex here. please see if this satisfy your requirement
also add more test cases (at least 5). (letters, numbers, spaces, multiple dots, special characters etc)
https://regexr.com/7c9mc

![image](https://user-images.githubusercontent.com/12880059/232780283-781f1d0c-5ec7-41d6-a0da-716104f5cf3c.png)